# HG changeset patch
# User Rik <rik@octave.org>
# Date 1425844521 25200
#      Sun Mar 08 12:55:21 2015 -0700
# Node ID 57379fcc1aecb86c25f4456327ba2168b457e2b8
# Parent  de0ac3d6ef595c74e4a71767f9348aeb3d539b4c
doc: Update title page to Edition 4 of manual.  Update copyright.

* octave.texi: Update title page to Edition 4 of manual.  Update copyright.

diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -84,18 +84,18 @@ a permission notice identical to this on
 Permission is granted to copy and distribute translations of this
 manual into another language, under the above conditions for
 modified versions.
 @end ifnottex
 
 @titlepage
 @title GNU Octave
 @subtitle A high-level interactive language for numerical computations
-@subtitle Edition 3 for Octave version @value{VERSION}
-@subtitle December 2013
+@subtitle Edition 4 for Octave version @value{VERSION}
+@subtitle March 2015
 @sp 2
 @multitable @columnfractions 0.4 0.025 0.65
 @item
 @flushright @image{octave_logo,2.1in}
 @end flushright
 @tab
 @c this is a spacer column
 @tab
@@ -104,19 +104,19 @@ modified versions.
 @end multitable
 @author John W. Eaton
 @author David Bateman
 @author S@o{}ren Hauberg
 @author Rik Wehbring
 @page
 @vskip 0pt plus 1filll
 Copyright @copyright{} 1996, 1997, 1999, 2000, 2001, 2002, 2005, 2006,
-2007, 2011, 2013 John W. Eaton.
+2007, 2011, 2013, 2015 John W. Eaton.
 
-This is the third edition of the Octave documentation, and is consistent
+This is the fourth edition of the Octave documentation, and is consistent
 with version @value{VERSION} of Octave.
 
 Permission is granted to make and distribute verbatim copies of
 this manual provided the copyright notice and this permission notice
 are preserved on all copies.
 
 Permission is granted to copy and distribute modified versions of this
 manual under the conditions for verbatim copying, provided that the entire
